include/EVT/dev/IWDG.hpp

```diff
@@ -9,6 +9,11 @@ namespace EVT::core::DEV {
  */
 class IWDG {
 public:
+    /**
+     * Initializes and activates the watchdog timer.
+     */
+    virtual void init() = 0;
```

Comment:
Why is this initialized to 0? Is it because it is virtual and get's implemented by individual boards?

Reply:
Yes, that's the way C++ likes virtual methods to be initialized, as I understand it